app/grandchallenge/reader_studies/forms.py

```diff
@@ -504,9 +570,18 @@ class Meta:
 
 
 class CategoricalOptionForm(ModelForm):
-    def __init__(self, *args, **kwargs):
+    def __init__(self, *args, is_editable, **kwargs):
         super().__init__(*args, **kwargs)
+
         self.fields["title"].label = False
+        self.fields["title"].required = True
```

Comment:
This doesn't seem to work. I can add an empty option.

Reply:
Are you sure? There is an empty option by default. Does it actually end up in the db?

Reply:
I cannot replicate this, how did you do it?

Reply:
It's controlled by the `extra` argument to the formset factory https://github.com/comic/grand-challenge.org/blob/00171b53cd1b23b3249be18bf355f57b4122b070/app/grandchallenge/reader_studies/forms.py#L596. I did set that to zero but turned it back to 1 as then you get an empty form to start with the save the user a click. The `hideLastAddForm` option then deleted it if there was no prior data but I didn't add back that functionality. Probably the easiest thing is to set `extra` to zero and then see if you prefer that.

Reply:
Not a strong preference for either or: when there already are options, I prefer if the empty option is not there. When there are no options to begin with, I do prefer the empty one because it makes it more obvious that you need to fill something in there. So yeah, leave it as is.

Reply:
I have made a PR that addresses this (#3453), please test it out extensively to see if it still performs as expected! I have targeted this branch so can be merged here if it's okay.